x/concentrated-liquidity/lp.go

```diff
@@ -64,15 +64,15 @@ func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 
 	// update pool state as well only if we're injecting liquidity at the current tick of the pool
 	if pool.CurrentTick.LT(sdk.NewInt(lowerTick)) {
-		amtDenom0 = calcAmount0Delta(liquidity, sqrtRatioLowerTick, sqrtRatioUpperTick).RoundInt()
+		amtDenom0 = calcAmount0Delta(liquidity, sqrtRatioLowerTick, sqrtRatioUpperTick, false).RoundInt()
```

Comment:
As discussed offline, am going to revert the rounding until we can create a test case that actually causes an overflow

Reply:
Further discussion offline, rounding both protects from inf loop as well as rounding up on amountIn to prevent removing more from the pool than expected.